shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -554,7 +555,7 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(dstFile.getAbsolutePath() + randomSuffix);
+    File tmpDst = new File(Name.HOME + randomSuffix);
```

Comment:
if users set the dstPath is `/home/username`
`File tmpDst = new File(dstFile.getAbsolutePath() + randomSuffix);` will do not have the permission to write.